build.gradle

```diff
@@ -29,6 +29,8 @@ oss {
     description 'Java client library for the Auth0 platform.'
     baselineCompareVersion '1.27.0'
     skipAssertSigningConfiguration true
+    enableJava8Testing true
+    enableJava17Testing true
```

Comment:
As mentioned on the plugin PR, it might be easier if we could configure the plugin with the versions to run tests in with one config point, like `testJavaVersions '8, 11, 7` or something like that. And then the plugin can run tests using those versions, without the lib needing to specify multiple lines or only enable certain versions.

Reply:
Fixed this in the latest commit +1